modules/cloud-run-v2/service.tf

```diff
@@ -15,14 +15,15 @@
  */
 
 resource "google_cloud_run_v2_service" "service" {
-  count        = var.create_job ? 0 : 1
-  provider     = google-beta
-  project      = var.project_id
-  location     = var.region
-  name         = "${local.prefix}${var.name}"
-  ingress      = var.ingress
-  labels       = var.labels
-  launch_stage = var.launch_stage
+  count          = var.create_job ? 0 : 1
+  provider       = google-beta
+  project        = var.project_id
+  location       = var.region
+  name           = "${local.prefix}${var.name}"
+  ingress        = var.ingress
+  labels         = var.labels
+  launch_stage   = var.launch_stage
+  encryption_key = var.encryption_key
```

Comment:
For symmetry, can we add this right away to job, as both resources support CMEK?

Reply:
Yes, we can.

Reply:
I don't see the file changed, did you forgot to commit that change?